troubleshooting.md

```diff
@@ -104,7 +104,18 @@ learning.puppetlabs.vm:8140` generally indicates that the `pe-puppetdb` service
 is down. Again, refer to the section above for instructions on checking and
 restarting PE services.
 
-### I cannot import the OVA
+When triggering a Puppet run via the `puppet job` tool, you may see an error
+similar to `Error running puppet on pasture.puppet.vm: pasture.puppet.vm is not
+connected to the PCP broker. A node connects to the PCP broker on its initial
```

Comment:
Should there be a closing backtick on this line to indicate the end of the error message? @kjhenner